_posts/2024-07-16-17.0.0-release.md

```diff
@@ -0,0 +1,176 @@
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 17.0.0 release. This covers
+over 3 months of development work and includes [**331 resolved issues**][1]
+on [**529 distinct commits**][2] from [**92 distinct contributors**][2].
+See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 16.0.0 release, Dane Pitkin has been invited to be committer.
+No new members have joined the Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## C Data Interface notes
+
+ - `ArrowDeviceArrayStream` can now be imported and exported (GH-40078)
+## Arrow Flight RPC notes
+
+- Flight SQL was formally stabilized (GH-39204).
+- Flight SQL added a bulk ingestion command (GH-38255).
+- The JDBC Flight SQL driver now accepts "catalog" as a connection parameter (GH-41947).
+- "Stateless" prepared statements are now supported (GH-37220, GH-41262).
+- Java added `FlightStatusCode.RESOURCE_EXHAUSTED` (GH-35888).
+- C++ has some basic support for logging with OpenTelemetry (GH-39898).
+
+## C++ notes
+
+- Half-float values can now be parsed and formatted correctly (GH-41089).
+- Record batches can now be converted to row-major tensors, not only column-major (GH-40866).
+- The CSV writer is now able to write large string arrays that are larger than
+  2 GiB (GH-40270).
+- A possible invalid memory access in `BooleanArray.true_count()` has been fixed (GH-41016).
+- A new method `FlattenRecursively` allows recursive nesting of list and
+  fixed-size list arrays (GH-41055).
+- The scratch space in some `Scalar` subclasses is now immutable. This is required
+  for proper concurrent access to `Scalar` instances (GH-40069).
+- Calling the `bit_width` or `byte_width` method of an extension type now defers
+  to the underlying storage type (GH-41353).
+- Fixed a bug where `MapArray::FromArrays` would behave incorrectly if the given
+  offsets array has a non-zero offset (GH-40750).
+- `MapArray::FromArrays` now accepts an optional null bitmap argument
+  (GH-41684).
+- The `ARROW_NO_DEPRECATED_API` macro was unused and has been removed (GH-41343).
+- Building with libc++ and C++20 enabled has been fixed (GH-43095).
+- mimalloc is now preferred over jemalloc as the default memory pool (GH-43254).
```

Comment:
Isn't https://github.com/apache/arrow/issues/43254 mark as 18.0.0?

Reply:
Oops, yes, you're probably right.